### PR TITLE
Automation tests for activity tab

### PR DESCRIPTION
**Pull request by lydia-karungi**
#Check Export filter
#Check Date range & Type
# :books: Linear Ticket :books:

### CODE REVIEW

**Comment by beemi** [comment]
@lydia-karungi  it would be good to write some tests to validate CSV data?
*  do any SEND
* then download CSV from activity
* Then validate the CSV data after downloaded

**Comment by lydia-karungi** [comment]
> @lydia-karungi it would be good to write some tests to validate CSV data?
> 
> * do any SEND
> * then download CSV from activity
> * Then validate the CSV data after downloaded

@beemi All comments have been worked on, kindly review again

File: test/e2e/activity_page.spec.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+const testUtil = new TestUtil()
+const overviewPage = new OverviewPage()
+const homePage = new HomePage()
+const passwordPage = new PasswordPage()
+const sendPage = new SendPage()
+
+let browser, page
+
+describe.only('Activity section["MAINNET"]', async () => {
```
Comment (beemi) [inline test/e2e/activity_page.spec.js:18]: can u remove .only

File: test/e2e/activity_page.spec.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    // Enter seed words and submit
+    await homePage.EnterSeedWords(page)
+
+    // Create a password & submit
+    await passwordPage.SubmitPasswordDetails(page)
+
+    // overview page
+    await overviewPage.CloseWhatsNewModal(page)
+    await overviewPage.HasOverviewPageLoaded(page)
+
```
Comment (beemi) [inline test/e2e/activity_page.spec.js:39]: i think better to switch to testnet for this test

**Review by beemi — CHANGES_REQUESTED** [review]

**Review by beemi — APPROVED** [review]

**Pull request merged**